<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1827</width>
    <height>831</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MolonaViz</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1827</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStudy">
    <property name="title">
     <string>Study</string>
    </property>
    <addaction name="actionCreate_Study"/>
    <addaction name="actionOpen_Study"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Study"/>
   </widget>
   <widget class="QMenu" name="menuPoint">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Point</string>
    </property>
    <addaction name="actionImport_Point"/>
    <addaction name="actionOpen_Point"/>
    <addaction name="actionRemove_Point"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionSwitch_To_Tabbed_View"/>
    <addaction name="actionSwitch_To_SubWindow_View"/>
    <addaction name="actionSwitch_To_Cascade_View"/>
    <addaction name="separator"/>
    <addaction name="actionData_Points"/>
    <addaction name="actionSensors"/>
    <addaction name="actionApplication_Messages"/>
   </widget>
   <widget class="QMenu" name="menuMolonaViz">
    <property name="title">
     <string>MolonaViz</string>
    </property>
    <widget class="QMenu" name="menuPreferences">
     <property name="title">
      <string>Preferences</string>
     </property>
     <widget class="QMenu" name="menuTemperature">
      <property name="title">
       <string>Temperature</string>
      </property>
      <addaction name="actionTempToKelvin"/>
      <addaction name="actionTempToCelsius"/>
     </widget>
     <addaction name="menuTemperature"/>
    </widget>
    <addaction name="actionAboutMolonaViz"/>
    <addaction name="menuPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuitMolonaViz"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpenUserguideFR"/>
   </widget>
   <widget class="QMenu" name="menuLabo">
    <property name="title">
     <string>Labo</string>
    </property>
    <addaction name="actionImport_Labo"/>
   </widget>
   <addaction name="menuMolonaViz"/>
   <addaction name="menuStudy"/>
   <addaction name="menuPoint"/>
   <addaction name="menuLabo"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockDataPoints">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Data Points</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeView" name="treeViewDataPoints">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockSensors">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Current lab: </string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Pressure Sensors</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeView" name="treeViewPressureSensors">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Shafts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeView" name="treeViewShafts">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Thermometers</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeView" name="treeViewThermometers">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockMessages">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Application Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTextEdit" name="textEditApplicationMessages"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionCreate_Study">
   <property name="text">
    <string>Create Study</string>
   </property>
  </action>
  <action name="actionOpen_Study">
   <property name="text">
    <string>Open Study</string>
   </property>
  </action>
  <action name="actionClose_Study">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Study</string>
   </property>
  </action>
  <action name="actionCreateStudy2">
   <property name="text">
    <string>CreateStudy2</string>
   </property>
  </action>
  <action name="actionImport_Point">
   <property name="text">
    <string>Import Point</string>
   </property>
  </action>
  <action name="actionOpen_Point">
   <property name="text">
    <string>Open Point</string>
   </property>
  </action>
  <action name="actionRemove_Point">
   <property name="text">
    <string>Remove Point</string>
   </property>
  </action>
  <action name="actionSwitch_To_Tabbed_View">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Switch To Tabbed View</string>
   </property>
  </action>
  <action name="actionSwitch_To_SubWindow_View">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Switch To SubWindow View</string>
   </property>
  </action>
  <action name="actionAboutMolonaViz">
   <property name="text">
    <string>About MolonaViz</string>
   </property>
  </action>
  <action name="actionQuitMolonaViz">
   <property name="text">
    <string>Quit MolonaViz</string>
   </property>
  </action>
  <action name="actionData_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="actionSensors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sensors</string>
   </property>
  </action>
  <action name="actionApplication_Messages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Application Messages</string>
   </property>
  </action>
  <action name="actionTempToKelvin">
   <property name="text">
    <string>Kelvin</string>
   </property>
  </action>
  <action name="actionTempToCelsius">
   <property name="text">
    <string>Celsius</string>
   </property>
  </action>
  <action name="actionSwitch_To_Cascade_View">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Switch To Cascade View</string>
   </property>
  </action>
  <action name="actionOpenUserguideFR">
   <property name="text">
    <string>Open Userguide (FR)</string>
   </property>
  </action>
  <action name="actionImport_Labo">
   <property name="text">
    <string>Import Labo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
